egs/swbd/run.sh

```diff
@@ -57,6 +57,18 @@ for order in 3 4 5; do
     format_arpa_lm.py data/lm_20k_${order}_prune${threshold} | gzip -c > data/arpa/20k_${order}gram_prune${threshold}.arpa.gz
 
   done
+
+# example of pruning by size.
```

Comment:
I also found this bug just now. sorry for that. i already fixed it, and created a pull request. you can test it after merged.